source/class/qx/ui/form/RadioGroup.js

```diff
@@ -522,7 +522,26 @@ qx.Class.define("qx.ui.form.RadioGroup",
 
       if (value) {
         value.set(groupedProperty, true);
+
+        // If Group is focused, the selection was changed by keyboard. Switch focus to new value
```

Comment:
Enhancement, it seems?

Reply:
Kinda needed. Radio Selection could change without focus moving, resulting in no reading out to the screenreader